src/renderer/browser/util/parseURL.js

```diff
@@ -1,7 +1,12 @@
-import { trim } from 'lodash';
+import { filter, isEmpty, trim, trimEnd, endsWith } from 'lodash';
+
+import { PROTOCOLS, CUSTOM_PROTOCOLS } from '../values/protocols';
 
 export default function parseURL(query) {
-  const trimmedQuery = trim(query);
-  const formattedQuery = trimmedQuery.includes('://') ? trimmedQuery : `nos://${trimmedQuery}`;
-  return new URL(formattedQuery);
+  const trimmedQuery = trimEnd(trim(query), '/').split('://').pop();
+
+  const tld = filter(CUSTOM_PROTOCOLS, (protocol) => endsWith(trimmedQuery, protocol.tld));
+  const tldToApply = isEmpty(tld) ? PROTOCOLS.http : tld[0].protocol;
```

Comment:
This is referencing `PROTOCOLS.http`, but it looks like it should be `PROTOCOLS.HTTP`.